charts/newrelic-infrastructure-v3/values.yaml

```diff
@@ -143,6 +143,10 @@ controlPlane:
   # -- Run Control Plane scraper with `hostNetwork` even if `privileged` is set to false.
   # `hostNetwork` is required for most control plane configurations, as they only accept connections from localhost.
   unprivilegedHostNetwork: false
+  # -- Deployment strategy when `kind` is `DaemonSet`.
+  # @default -- `type: Recreate`
+  strategy:
+    type: Recreate
```

Comment:
I would delete the `strategy` and `updateStrategy` and leave them empty to the cluster to choose those values.

I could leave the `strategy` as `Recreate` is not the default strategy type...

Reply:
As we have discussed, defaulting to anything other than `Recreate` will result in a broken experience if `hostNetwork` is in use.